main.go

```diff
@@ -42,33 +27,35 @@ func main() {
 	// to keep track of original target sizes during rolling updates
 	originalDesired := map[string]int64{}
 
-	checkDelay, err := getDelay()
-	if err != nil {
-		log.Fatalf("Unable to get delay: %s", err.Error())
-	}
-
 	// infinite loop
 	for {
-		err := adjust(asgList, ec2Svc, asgSvc, readinessHandler, originalDesired)
+		err := adjust(
+			configs.KubernetesEnabled, configs.ASGS, ec2Svc, asgSvc,
+			readinessHandler, originalDesired, configs.OriginalDesiredOnTag,
+			configs.IncreaseMax, configs.Verbose, configs.Drain, configs.DrainForce,
+		)
 		if err != nil {
 			log.Printf("Error adjusting AutoScaling Groups: %v", err)
 		}
 		// delay with each loop
-		log.Printf("Sleeping %d seconds\n", checkDelay)
-		time.Sleep(time.Duration(checkDelay) * time.Second)
+		log.Printf("Sleeping %d seconds\n", configs.Interval)
```

Comment:
Is this correct? If someone put in a `time.Duration` of `2m`, when I put this as `"Sleeping %d seconds", configs.Interval`, what will it report?

Reply:
Good catch. I think I will change this to `"Sleeping %v\n", configs.Interval"` - While not as descriptive you will see "Sleeping 2m" or "Sleeping 30s", I believe we can get the full word out of `time` but it might be more work than worth it. Thoughts?

Reply:
Pushed that variant of the change for now but we can change it